index.js

```diff
@@ -1461,26 +1497,33 @@
   //. Curries the given quinary function.
   //.
   //. ```javascript
-  //. > const toUrl = S.curry5((protocol, creds, hostname, port, pathname) =>
+  //. > const toUrl = S.curry5 ((protocol, creds, hostname, port, pathname) =>
   //. .   protocol + '//' +
-  //. .   S.maybe('', _ => _.username + ':' + _.password + '@', creds) +
+  //. .   S.maybe ('') (S.flip (S.concat) ('@')) (creds) +
   //. .   hostname +
-  //. .   S.maybe('', S.concat(':'), port) +
+  //. .   S.maybe ('') (S.concat (':')) (port) +
   //. .   pathname
   //. . )
   //.
-  //. > toUrl('https:')(S.Nothing)('example.com')(S.Just('443'))('/foo/bar')
-  //. 'https://example.com:443/foo/bar'
-  //.
-  //. > toUrl('https:', S.Nothing, 'example.com', S.Just('443'), '/foo/bar')
+  //. > toUrl ('https:') (S.Nothing) ('example.com') (S.Just ('443')) ('/foo/bar')
   //. 'https://example.com:443/foo/bar'
   //. ```
-  function curry5(f, v, w, x, y, z) {
-    return f(v, w, x, y, z);
+  function curry5(f) {
+    return function(v) {
+      return function(w) {
+        return function(x) {
+          return function(y) {
+            return function(z) {
+              return f (v, w, x, y, z);
```

Comment:
For some reason this makes me happy. :D

Reply:
I was typing a similar comment to `curry`:

> So blissfully transparent! :D

...but then I scrolled down and saw `curry2`, which made me happier, so I removed the comment to move it over there. But then I scrolled down further to find `curry3`. Woah, how cool is this going to get? I scrolled all the way to `curry5` and found your comment. :)

Reply:
:rofl: